documentation/src/main/markdown/packagemgrs/nuget.md

```diff
@@ -91,16 +91,17 @@ The NuGet Project Native Inspector derives packages (dependencies) from project
 "*.rproj"
 ````
 
-The NuGet Solution Native Inspector runs if one or more solution (.sln) files are found. Any package references and versions in the solution's Directory.Build.props will be included for each project under the solution.
+The NuGet Solution Native Inspector runs if one or more solution (.sln) files are found. It will now support Central Package Management and include any package versions and global package references mentioned under Directory.Packages.props files mentioned in .sln file. Any package references and versions in the solution's Directory.Build.props will be included for each project under the solution.
```

Comment:
Just so we don't have a statement that has a time factor, (i.e. "now"), and reduce the existing last line redundancy.
Something like:
The NuGet Solution Native Inspector runs if one or more solution (.sln) files are found. Central Package Management is supported to include any package versions and global package references mentioned under `Directory.Packages.props` files indicated the (.sln) file for each project under the solution.